module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatter.php

```diff
@@ -377,6 +377,50 @@ protected function getLink($value, $options)
         return false;
     }
 
+    /**
+     * Render standard and alternative fields together.
+     *
+     * @param mixed $data    Data to render
+     * @param array $options Rendering options.
+     *
+     * @return string
+     */
+    protected function renderCombineAlt(
+        $data,
+        array $options
+    ) {
+        // Determine the rendering method to use, and bail out if it's illegal:
+        $method = empty($options['combineAltRenderType'])
+            ? 'renderSimple' : 'render' . $options['combineAltRenderType'];
```

Comment:
Is it worth adding a test case to cover combineAltRenderType? I don't think I see one...

Reply:
Added one for renderType "RecordDriverTemplate"